tsdb/points.go

```diff
@@ -604,7 +608,8 @@ func scanFieldValue(buf []byte, i int) (int, []byte) {
 			break
 		}
 
-		if buf[i] == '"' {
+		// If we see a double quote, makes sure it is not escaped
+		if buf[i] == '"' && buf[i-1] != '\\' {
```

Comment:
Technically, yes, if you call `scanFieldValue(buf, 0)` directly then it can blow up but `i` can't be zero during parsing because the we need to scan the the field name first and it must be non-zero in length so the function is always called with a non-zero value for `i`.  I could special case it here too but didn't think it was necessary since the func is private and only called in one place.